src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs

```diff
@@ -165,16 +170,15 @@ protected virtual bool IsNewHeadAlignedWithChain(Block newHeadBlock, ForkchoiceS
 
         if (!blockInfo.WasProcessed)
         {
-            if (_blockTree.IsOnMainChainBehindHead(newHeadBlock))
+            if (!IsNewHeadAlignedWithChain(newHeadBlock, forkchoiceState, out ResultWrapper<ForkchoiceUpdatedV1Result>? errorResult))
```

Comment:
`IsNewHeadAlignedWithChain` is cryptic naming, can we name it better? Maybe would be easier if we would flip the bool semantics

Reply:
already sounds easier to me